msm/mycroft_skills_manager.py

```diff
@@ -207,12 +226,16 @@ def update(self, skill=None, author=None):
                 if entry:
                     entry['updated'] = time.time()
 
+    @save_skills_data
     def apply(self, func, skills):
         """Run a function on all skills in parallel"""
 
         def run_item(skill):
             try:
-                func(skill)
+                if 'save_skills_data' in dir(func):
+                    func(skill, __msm_write=False)
```

Comment:
Does the `__msm_write=False` still refer to anything?

Reply:
No, you're right. That's a remnant from my first approach. Removing